.travis.yml

```diff
@@ -12,32 +12,28 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 7.1
+    - php: 7.3
       env:
         - DEPS=lowest
-    - php: 7.1
+    - php: 7.3
       env:
         - DEPS=latest
         - TEST_COVERAGE=true
         - STATIC_ANALYSIS=true
-    - php: 7.2
-      env:
-        - DEPS=lowest
-    - php: 7.2
-      env:
-        - DEPS=latest
-    - php: 7.3
+    - php: 7.4
       env:
         - DEPS=lowest
-    - php: 7.3
+    - php: 7.4
       env:
         - DEPS=latest
-    - php: 7.4
+    - php: nightly
       env:
         - DEPS=lowest
-    - php: 7.4
+    - php: nightly
       env:
         - DEPS=latest
```

Comment:
for nightly, you can add `COMPOSER_ARGUMENTS="$COMPOSER_ARGUMENTS --ignore-platform-reqs` so it can install all dependencies even if these are not yet supporting PHP 8.0.

Reply:
it's already set for all php version: https://github.com/laminas/laminas-di/blob/3.2.x/.travis.yml#L46

Reply:
As you can see, build on PHP 8 is still failing. This is due to the fact that the `COMPOSER_ARGUMENTS` do not gobally ignore platform reqs. To fix this, you can drop `--ignore-platform-reqs` from line 46 and instead, add it to the nightly envs. After this, travis should pass.